routes/project/region.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch(:project_prefix, "region") do |r|
+    r.get true do
+      puts "project: #{@project.path}"
+      regions = @project.aws_location_credentials
+
+      if api?
+        Serializers::Region.serialize(regions)
+      else
+        @regions = Serializers::Region.serialize(regions)
+        view "region/index"
+      end
+    end
+
+    r.post true do
+      puts "geldieem project: #{@project.path}"
+      authorize("Region:create", @project.id)
+      required_parameters = ["name", "aws_region_name", "aws_access_key", "aws_secret_key"]
+      request_body_params = validate_request_params(required_parameters)
+      unless @project
+        if api?
+          fail ValidationError.new("Project not found")
+        else
+          r.redirect "region/create"
+        end
+      end
+
+      if @project.accounts_dataset.where(Sequel[:accounts][:id] => current_account_id).empty?
```

Comment:
Can we reach a point where this is true?  This looks like it duplicates a check in `routes/project.id`?